platform-coop-toolkit/coop-library
# fix: resolve PHP warnings, only show 'Applied filters' if needed (fix #204)

## Description

This PR resolves some PHP warnings I was seeing and hides the 'Applied filters' label unless needed.

## Steps to test

1. Perform a search.

**Expected behavior:** Verify that 'Applied filters' is only shown if filters are applied to the search results.

## Additional information

Not applicable.

## Related issues

- Resolves #204.
